src/Explorer/Tabs/DocumentsTab.ts

```diff
@@ -304,6 +307,16 @@ export default class DocumentsTab extends TabsBase {
         return true;
       }),
     };
+
+    this.copliotEntityButton = {
+      enabled: ko.computed<boolean>(() => {
+        return userContext.features.enableCopilot;
```

Comment:
You don't have to make the enabled and visible flag a knockout observable. The value of `userContext.features.enableCopilot` will not change during the same session.

Reply:
Just used the feature flag remove observable.